### PR TITLE
Updated HPC Installation Section

### PR DESCRIPTION
**Pull request by feathern**
This PR adds instructions for compilation on the NASA Pleiades system.  It also adds a Rayleigh/job_scripts directory with subdirectories for Stampede2 and Pleiades, each of which contains a sample jobscript along with commands needed to compile the code.    That way, everything is easily accessible once the repository is cloned.  I removed the TACC jobscript from the main documentation to streamline things and avoid too much duplication of information.  I think the installation instructions are useful to retain, even if they're summarized in the jobscripts.

Happy to iterate on this, change directory names etc.

### CODE REVIEW

**Comment by feathern** [comment]
Also, as a reminder, the draft text can be viewed here: https://rayleigh-documentation--417.org.readthedocs.build/en/417/

**Review by gassmoeller — APPROVED** [review]
Looks good, I found a typo and have a few small comments which are not very important. Feel free to address or ignore and then merge.

File: job_scripts/NASA_Pleiades/pleiades_script.sh [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#PBS -S /bin/bash
+#PBS -q long                # 'long' queue allows jobs up to 5 days (use 'normal' for 8 hours or less).
+#PBS -j oe
+#PBS -W group_list=s1234    # account number
+#PBS -m e
+#PBS -N nrho5_ff_5          # job name
+#PBS -l walltime=120:00:00  # run for 120 hours
+#PBS -l select=74:ncpus=28:mpiprocs=28:model=bro  # Request Broadwell nodes.  28 MPI ranks per node.
```
[the remark this answers is not in the document]
Comment (feathern) [reply job_scripts/NASA_Pleiades/pleiades_script.sh]: Good catch -- just made a note in the recent commit.

File: job_scripts/NASA_Pleiades/pleiades_script.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+module purge
+module load comp-intel
+module load mpi-hpe
+export OMP_NUM_THREADS=1
+
+# Link the Rayleigh executables from your home directory (modify the path below accordingly)
+ln -s /home4/nfeather/Ra_share/Rayleigh/bin/* .
+sleep 10
+
+# Run the code
```
[the remark this answers is not in the document]
Comment (feathern) [reply job_scripts/NASA_Pleiades/pleiades_script.sh:19]: Agreed - modified.

File: job_scripts/TACC_Stampede2/stampede2_jobscript.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+module list
+
+# Launch MPI code...
+
+export OMP_NUM_THREADS=1
+# Replace -n X with correct number of MPI ranks, or remove to use all ranks
+# requested on the nodes above.
+
+ibrun -n 48 ./rayleigh.opt
```
[the remark this answers is not in the document]
Comment (feathern) [reply job_scripts/TACC_Stampede2/stampede2_jobscript.sh:20]: I'll leave this one as-is.    I could imagine someone running with a radial resolution that doesn't factor well into powers of 3 and 2, so they may want to run with (say) 46 cores.  In that case, they would need to be explicit.

**Comment by feathern** [comment]
Thanks for the review Rene -- merging.

**Pull request merged**